recipes/bamutil/build.sh

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 
 VERSION=1.0.14
-wget -O libStatGen-v$VERSION.tar.gz https://github.com/statgen/libStatGen/archive/v$VERSION.tar.gz
+curl -o libStatGen-v$VERSION.tar.gz -L https://github.com/statgen/libStatGen/archive/v$VERSION.tar.gz
```

Comment:
Can you also add
```bash
openssl sha256 libStatGen-v$VERSION.tar.gz | grep 70a504c5cc4838c6ac96cdd010644454615cc907df4e3794c999baf958fa734b
```

Reply:
This is nice! I thought about that the file isn't checked, but didn't think of a way to do it... perhaps a bit too focused on getting it to work

Reply:
No worries, there are a couple of recipes don't yet check those downloads. But in future, once we make `conda-build 3` available to Bioconda, we can use multiple source files in `meta.yaml` and can remove those "manual" downloads and checks :wink: